Gemfile

```diff
@@ -20,6 +20,7 @@ gem "sass-rails", "~> 5.0"
 gem "sdoc", "~> 0.4.0", group: :doc
 gem "uglifier", ">= 1.3.0"
 gem "unicorn"
+gem "reform"
```

Comment:
Are you suggesting we should be using `reform` for forms? In the service we use custom ActiveModel classes as form objects.